### ISSUE TITLE
Part of the instructions destroy the path to vim, ls, and other required tools

### ISSUE THREAD

**Issue by JamieL22** [MicrosoftDocs/sql-docs#7288]
I am new to Linux but after building the RHEL7 server several times, it would appear there is an error with the path command.   I'm not sure what that error is but when it runs, it destroys the path provided by the vendor.   Errors like "bash: sed: command not found"  and mostly any commandline entry in RHEL7 terminal will only return that error "command not found"   I suspect it is a path error so my thought is, the path command listed in the article is incorrect.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3b543b57-f397-779d-6fe4-1bc2ecd3bdca
* Version Independent ID: e274a0f3-ec74-e809-f4d3-515782b29a92
* Content: [RHEL: Install SQL Server on Linux - SQL Server](https://docs.microsoft.com/en-us/sql/linux/quickstart-install-connect-red-hat?view=sql-server-ver15)
* Content Source: [docs/linux/quickstart-install-connect-red-hat.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/linux/quickstart-install-connect-red-hat.md)
* Product: **sql**
* Technology: **linux**
* GitHub Login: @VanMSFT
* Microsoft Alias: **vanto**

**Comment by WilliamAntonRohm** [comment]
@JamieL22 -- Jamie, thank you for your feedback.  Please consider these resources:
- [Visual Studio on Microsoft Q&A](https://docs.microsoft.com/answers/products/vs) 
- [Visual Studio MSDN Forum](https://social.msdn.microsoft.com/Forums/vstudio) 
- [Stack Overflow](https://stackoverflow.com/search?q=%22visual+studio%22)  
- [Reddit](https://www.reddit.com/r/VisualStudio)

@VanMSFT -- Van, please look into this issue.

**Comment by VanMSFT** [comment]
Hi @JamieL22 - Thanks for the feedback. The command to add `/opt/mssql-tools/bin/` to the PATH environment variable is correct. Make sure that you have not overwritten the environment variable.

**Comment by JamieL22** [comment]
Van To,

I was able to get it to work the third time I installed RHEL7 server by simply using the “ln” command to join the /opt/mssql-tools/bin/* to the /usr/local/bin/  I was not able to make it work using the steps provided even when starting from scratch again with a new copy of SQL Server running on RHEL7.  I suspect there is a security issue though because it provides all users with a route to the /opt/mssql-tools/bin folder.

Thanks for checking though.

Jamie
